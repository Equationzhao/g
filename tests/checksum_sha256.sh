output="$(g -term-width 200 --no-config --no-update --icons --checksum -ca sha256 tests/test_data )"
echo "$output" | diff - tests/checksum_sha256.stdout
