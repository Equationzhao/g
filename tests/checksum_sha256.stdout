                                                                 [0;94m ?.dir[0m                                                      
[4m5c1a09fc486bfc1bfae26192b286909f7f190602a71ebde95d44c0281f12765f[0m [0;37m '   space file '[0m                                           
[4me3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855[0m [0;37m 0.1.112[0m                                                    
[4me3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855[0m [0;37m 0.1.13[0m                                                     
[4me3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855[0m [0;37m 0.1.2[0m                                                      
[4me3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855[0m [0;37m 0.2.10[0m                                                     
[4me3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855[0m [0;37m 0.2.2[0m                                                      
[4me3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855[0m [0;37m 0.3.0[0m                                                      
[4me3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855[0m [0;37m 0.3.84[0m                                                     
[4me3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855[0m [0;37m 1.0.1[0m                                                      
[4me3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855[0m [0;37m 1.4.1[0m                                                      
[4m0a23afe0bd19853e6c11f581cc6c59afe1fc6b208b471011cf2338a91e4094ba[0m [38;5;153m 1.txt[0m                                                      
[4meb62afab881c4ed48b3101a299877e1eb72b0aca80307813ef7523db347f71ff[0m [0;37m 10.json[0m                                                    
[4m11261b56992fd8b3c786eba95aa570a01f50c68c2c9f3ce9785f1d5952693f06[0m [0;37m 100KiB[0m                                                     
[4m335d62170d598be20264cb313206dea0e67f364cd7b68349bd3718110231a31e[0m [0;37m 11.xml[0m                                                     
[4m3d4f6cb3de1b1d7a3e50b13bb2ab41cde95992a147928aff113a8a2ab48c7845[0m [0;37m 12.yml[0m                                                     
[4mcb504c91ef546f76741fb8fd4c13b1f97e4b5ce2a9d78afa545fb6ec799e06c2[0m [0;93m 13.md[0m                                                      
[4me3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855[0m [0;95m 14.gif[0m                                                     
[4me3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855[0m [0;37m 2.19.1[0m                                                     
[4me3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855[0m [0;37m 2.9.1[0m                                                      
[4maf30ec07dea168bbab031061c22ccdb6ee0035eb7242c87bfadbe89fa829ec91[0m [38;5;158m 2.c[0m                                                        
[4m7d618e173f7234b662969093cf5945b87ad61e6d454b7d73e0ea285c9548b323[0m [38;5;158m 3.cpp[0m                                                      
[4me3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855[0m [38;5;158m 4.py[0m                                                       
[4mdae3582ada2811d3eb64139f03dbd9dfaf0e668b1b5527c2f0912b648c88c7ad[0m [38;5;158m 5.java[0m                                                     
[4me3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855[0m [0;92m 6.sh[0m                                                       
[4mf9bc6f8b5a69c69e3cc5f1a9c15d24fdb9f0307cb4c297c796d6894e8738b690[0m [38;5;158m 7.html[0m                                                     
[4m2069e2b8cccfe10d79865498ed25297c22059547209dc01c95331fdef3185f8d[0m [38;5;158m 8.css[0m                                                      
[4m35ec5bae0feeed66454c04ae9a6871da98d192c448315ca371a9cd8e8be82cca[0m [38;5;158m 9.js[0m                                                       
[4me3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855[0m [0;37m 9.js~[0m                                                      
[4m0a23afe0bd19853e6c11f581cc6c59afe1fc6b208b471011cf2338a91e4094ba[0m [0;35m 1.txt.link[0m[0;97m => [0m[38;5;153m/Users/equationzhao/g/tests/test_data/1.txt[0m  
